project-base/docker/php-fpm/docker-install-composer

```diff
@@ -0,0 +1,20 @@
+#!/bin/sh
+
+# copied from https://getcomposer.org/doc/faqs/how-to-install-composer-programmatically.md
+# option --filename=composer is used so it can be called without the .phar extension
+
+EXPECTED_SIGNATURE="$(wget -q -O - https://composer.github.io/installer.sig)"
```

Comment:
I remember some Backlog Grooming, where we talked about the installation of the latest composer (there is the US for it) and we decided not to. What made you change this decision?

Reply:
It's US-3775, let's ask the team about it.

Reply:
The internal voting was 5:1 in favor of up-to-date installation.